packages/common/mui/src/lib/line-chart/line-chart-tooltip/line-chart-tooltip.tsx

```diff
@@ -16,7 +32,10 @@ export const LineChartToolTip = (props: LineChartToolTipProps) => {
     <ToolTipBox>
       <Item>
         <H5 sx={{ fontSize: '12px', fontWeight: '600' }}>
-          {payload && payload.length > 0 ? payload[0]?.payload?.date : ''}
+          {payload && payload.length > 0
```

Comment:
Can we pass the timestamp in instead of using the monthMapping? For x-axis formatting in the line chart you can pass in a function to `tickFormatter` on the `xAxis` component:
https://github.com/notional-finance/notional-monorepo/blob/7bf9e74c17ea7a1b18e62e8204083248fd3c2ece/packages/common/mui/src/lib/area-chart/area-chart.tsx#L175-L177